docs/Changelog.md

```diff
@@ -8481,6 +8481,50 @@ This version of the operator has been available since version 9 of the default O
 <dd>Constrain input and output types to float tensors.</dd>
 </dl>
 
+### <a name="Cast-9"></a>**Cast-9**</a>
+
+  The operator casts the elements of a given input tensor to a data type
+  specified by the 'to' argument and returns an output tensor of the same size in
+  the converted type. The 'to' argument must be one of the data types specified
+  in the 'DataType' enum field in the TensorProto message. Litrals string 'NaN' 
+  and 'INF' are defined to be supported in a case-insensitive manner when cast from
```

Comment:
There are two `INF`, positive and negative. Which one does `INF` mean? If `INF' stands for positive inf, can we do `-INF` for negative inf? In addition, due to the existence of `-INF`, would it be nice to support `+INF`?

Reply:
Sure. Will clarify that